CMakeLists.txt

```diff
@@ -28,6 +28,15 @@ option(use_cppunittest "set use_cppunittest to ON to build CppUnitTest tests on
 option(run_traceability "run traceability tool (default is ON)" ON)
 option(run_reals_check "set run_reals_check to ON to run reals check (default is OFF)." OFF)
 
+if((NOT "${CMAKE_BUILD_TYPE}" STREQUAL "RelWithDebInfo") AND (NOT "${CMAKE_BUILD_TYPE}" STREQUAL "Debug"))
+    message(FATAL_ERROR "CMAKE_BUILD_TYPE must be specified and be precisely one of Debug or RelWithDebInfo")
+endif()
+
+set(CMAKE_CONFIGURATION_TYPES "${CMAKE_BUILD_TYPE}" CACHE STRING "" FORCE)
+
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY_DEBUG $ENV{BUILD_BINARIESDIRECTORY}/Debug)

```

Comment:
>BUILD_BINARIESDIRECTORY [](http://example.com/codeflow?start=46&length=23)

I know we verify this env var to exist in EBS repo, but I think this pushes the check all the way down to c-pal.

Let's make sure that the env variable exists at cmake time!

Reply:
Added check for it